tools/torch_quant/torch_quant/observer.py

```diff
@@ -116,6 +170,60 @@ def forward(self, x):
             return x
 
 
+class PerChannelMinMaxObserver(Observer):
+    def __init__(self, ch_axis=0, dtype: torch.dtype = torch.qint8,
+                 qscheme: torch.qscheme = torch.per_channel_symmetric, **kwargs) -> None:
+        super().__init__(dtype, qscheme, **kwargs)
+        self.register_buffer("min_val", torch.tensor([]))
+        self.register_buffer("max_val", torch.tensor([]))
+        self.register_buffer("scale", torch.tensor([]))
+        self.register_buffer("zero_point", torch.tensor([], dtype=torch.int32))
+        self.ch_axis = ch_axis
+
+    @property
+    def qparams(self) -> QParams:
+        return QParams(qscheme=self.qscheme, dtype=self.dtype,
+                       scale=self.scale,
+                       zero_point=self.zero_point)
```

Comment:
move scale, zero_point, and qparams() to base Observer class?

Reply:
done